IntroOOP/AbstractFactory/src/com/globant/abstractfactory/factory/FactoryProducer.java

```diff
@@ -0,0 +1,13 @@
+package com.globant.abstractfactory.factory;
+
+public class FactoryProducer {
+
+  public static SQLFactory getFactory(String factoryType) {
+    if (factoryType.equalsIgnoreCase("MySQL")) {
```

Comment:
This could throw a NullPointerException(NPE). Check [yoda conditions](https://knowthecode.io/yoda-conditions-yoda-not-yoda) to avoid this